middleware/service/source/manager/state.cpp

```diff
@@ -123,6 +123,14 @@ namespace casual
                   m_service = replacement;
             }
 
+            std::optional< std::string> Sequential::reserved_service() const
+            {
+               if( m_service != nullptr)
+                  return m_service->information.logical_name();
+
```

Comment:
nitpick, I would rather see `if( ! m_service)`. Why? more generic. If we change `m_service` to be, say an optional, it'll still work.

Reply:
` != nullptr` is perfectly fine though.